.circleci/config.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       - run: *merge-check
       - run: *apt-install
       - run: *tox-install
-      - run: tox -e build_docs
+      - run: tox -e build_docs -- -j=auto
```

Comment:
parallel docs :scream_cat:

Reply:
It didn't really do anything. Not sure if the setting is wrong or something else is up.